### PR TITLE
Dev -> Master for v2.0 release

### PR DESCRIPTION
**Pull request by bjlang**
# nf-core/atacseq pull request

Many thanks for contributing to nf-core/atacseq!

Please fill in the appropriate checklist below (delete whatever is not relevant).
These are the most common things requested on pull requests (PRs).

## PR checklist

- [x] This comment contains a description of changes (with reason)
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If necessary, also make a PR on the [nf-core/atacseq branch on the nf-core/test-datasets repo](https://github.com/nf-core/test-datasets/pull/new/nf-core/atacseq)
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [x] Make sure your code lints (`nf-core lint .`).
- [x] Documentation in `docs` is updated
- [x] `CHANGELOG.md` is updated
- [x] `README.md` is updated

**Learn more about contributing:** [CONTRIBUTING.md](https://github.com/nf-core/atacseq/tree/master/.github/CONTRIBUTING.md)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,143 +1,211 @@
...
-The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
-and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unpublished Version / DEV]
+
+### :warning: Major enhancements
+
+- Samplesheet format has changed from `group,replicate,fastq_1,fastq_2` to `sample,fastq_1,fastq_2,replicate`.
```
Comment (maxulysse) [inline CHANGELOG.md:10]: Is the order of importance?

**Comment by maxulysse** [comment]
I'm not sure I understand the logic of the folders for the blacklists:
```
assets/blacklists/v1.0/GENOME-blacklist.v1.bed
assets/blacklists/v2.0/GENOME-blacklist.v2.bed
assets/blacklists/v3.0/GENOME-blacklist.v3.bed
```
is the `vX.0` the same version as the `vX`?
Which versions of what are they referring to?

File: bin/bampe_rm_orphan.py [hunk 2]
```diff
@@ -17,17 +17,25 @@
 ############################################
 ############################################
 
-Description = 'Remove singleton reads from paired-end BAM file i.e if read1 is present in BAM file without read 2 and vice versa.'
+Description = (
+    "Remove singleton reads from paired-end BAM file i.e if read1 is present in BAM file without read 2 and vice versa."
+)
```
Comment (maxulysse) [inline bin/bampe_rm_orphan.py:22]: Is this linted with Black?

File: lib/WorkflowMain.groovy [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+
+class WorkflowMain {
+
+    //
+    // Citation string for pipeline
+    //
+    public static String citation(workflow) {
+        return "If you use ${workflow.manifest.name} for your analysis please cite:\n\n" +
+            "* The pipeline\n" +
+            "  https://doi.org/10.5281/zenodo.3240506\n\n" +
```
Comment (maxulysse) [inline lib/WorkflowMain.groovy:13]: This point to chipseq 2.0 with the name for the framework in the zenodo record
Could only find this record for atacseq: https://doi.org/10.5281/zenodo.2656367, which is 1.0, nothing for the other releases.
Any idea what happened there @nf-core/core

Comment (maxulysse) [reply lib/WorkflowMain.groovy:13]: nevermind, proper zenodo doi is in README: https://zenodo.org/record/6544493

Comment (maxulysse) [reply lib/WorkflowMain.groovy:13]:
```suggestion
            "  https://doi.org/10.5281/zenodo.2634132\n\n" +
```

File: CHANGELOG.md (hunk 1 above)
Comment (drpatelh) [reply CHANGELOG.md:10]: Yep, has to be in that order.

File: bin/bampe_rm_orphan.py (hunk 2 above)
Comment (drpatelh) [reply bin/bampe_rm_orphan.py:22]: Yep, tests seem to be passing for that on the PR.

File: lib/WorkflowMain.groovy (hunk 3 above)
Comment (drpatelh) [reply lib/WorkflowMain.groovy:13]: Well spotted! Will be fixed in a incoming PR.

Comment (maxulysse) [reply lib/WorkflowMain.groovy:13]: but then, we still should get rid of https://zenodo.org/record/2656368 which is a duplicate of https://zenodo.org/record/2634133

File: .gitignore [hunk 4]
```diff
@@ -3,6 +3,6 @@ work/
 data/
 results/
 .DS_Store
-tests/
 testing/
+testing*
```
Comment (JoseEspinosa) [inline .gitignore:7]: Just for me to know, what is the reason for this change in the template?

File: lib/WorkflowMain.groovy (hunk 3 above)
Comment (drpatelh) [reply lib/WorkflowMain.groovy:13]: Ah, I had issues with this when I first released the pipeline. Had to create one manually which we don't need now.

**Review by JoseEspinosa — APPROVED** [review]
LGTM! 
Though I might be a little bit biased :eyes:

File: .gitignore (hunk 4 above)
Comment (drpatelh) [reply .gitignore:7]: No idea!

**Review by maxulysse — APPROVED** [review]
LGTM
